src/DependencyInjection/SonataNotificationExtension.php

```diff
@@ -163,19 +161,17 @@ public function configureBackends(ContainerBuilder $container, $config)
     }
 
     /**
-     * @param array $config
+     * NEXT_MAJOR: remove this method, since all Doctrine mappings are already
+     *             described in Resources/config/doctrine/BaseMessage.orm.xml.
+     *
+     * @deprecated Since version 3.x, to be removed in 4.0.
+     *             EasyExtends should be used only for associations between entities.
      */
     public function registerDoctrineMapping(array $config)
     {
-        $collector = DoctrineCollector::getInstance();
-
-        $collector->addIndex($config['class']['message'], 'idx_state', [
-            'state',
-        ]);
-
-        $collector->addIndex($config['class']['message'], 'idx_created_at', [
-            'created_at',
-        ]);
+        @trigger_error(sprintf(
```

Comment:
IMHO we should remove this code without any further ado. Otherwise we will always throw an exception that the user can't fix.